config.json

```diff
@@ -78,6 +79,11 @@
     "RealtimeChannelID": ""
   },
   "Commands": [
```

Comment:
Is it worth renaming this to `ChatCommands`? Not sure how ambigous `Commands` is.

Reply:
I don't think "commands" is overly ambiguous. It'd work fine I believe.